src/resources/views/crud/columns/color.blade.php

```diff
@@ -0,0 +1,28 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['escaped'] = $column['escaped'] ?? true;
+    $column['text'] = $column['default'] ?? '#000000';
+
+    if($column['value'] instanceof \Closure) {
+        $column['value'] = $column['value']($entry);
+    }
+
+    if(is_array($column['value'])) {
+        $column['value'] = json_encode($column['value']);
+    }
+
+    if(!empty($column['value'])) {
+        $column['text'] = $column['value'];
+    }
+@endphp
+
+<span>
+    @includeWhen(!empty($column['wrapper']), 'crud::columns.inc.wrapper_start')
+        @if($column['escaped'])
+            <span style="background-color: {{ $column['text'] }}"></span> {{ $column['text'] }}
+        @else
+            <span style="background-color: {!! $column['text'] !!}"></span> {{ $column['text'] }}
```

Comment:
We show twice the `$column['text']`, but only escape the first time.

Reply:
@tabacitu suggest me to show HEX color code to copy. Please suggest?

Reply:
@munjaldevelopment `#{!! substr($column['text'], 1) !!}`

Reply:
This apply on style or in text?

Reply:
Hello @munjaldevelopment , from my tests we can display both with `{!! $column['text'] !!}}` because it does not escape `#`.